### PR TITLE
SAK-46968 Assignments: canSubmit now checks getUserSubmission

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-46968

### CODE REVIEW

File: assignment/impl/src/test/org/sakaiproject/assignment/impl/AssignmentServiceTest.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 import static org.mockito.ArgumentMatchers.anyCollection;
 import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.reset;
```
Comment (bgarciaentornos) [inline assignment/impl/src/test/org/sakaiproject/assignment/impl/AssignmentServiceTest.java:28]: I think this import is not being used

**Review by bgarciaentornos — APPROVED** [review]
LGTM, just one little thing

File: assignment/impl/src/test/org/sakaiproject/assignment/impl/AssignmentServiceTest.java (hunk 1 above)
Comment (adrianfish) [reply assignment/impl/src/test/org/sakaiproject/assignment/impl/AssignmentServiceTest.java:28]: You're right! thanks.

Comment (adrianfish) [inline assignment/impl/src/test/org/sakaiproject/assignment/impl/AssignmentServiceTest.java:28]:
```suggestion
```

**Pull request merged**